recipes/fandango/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fandango" %}
+{% set version = "15.10.3" %}
+{% set python_min = "3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/fandango-{{ version }}.tar.gz
+  sha256: a3a6fabe301c33d1715f267733b3a5e677de03d9b1e43e0111e7630402ecbeaf
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools    
+    - future    
+  run:
+    - python >={{ python_min }}
+    - future
+
+test:
+  imports:
+    - fandango
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+    - future
+    - setuptools
```

Comment:
future and setuptools aren't needed here.
(future is a run requirement so it will be installed automatically).

Reply:
ok, I will update that

Reply:
Still need to remove future and setuptools